expertsystem/io/_pdg.py

```diff
@@ -0,0 +1,129 @@
+"""Create a `.ParticleCollection` instance from PDG info."""
+
+from typing import Optional, Tuple
+
+from particle import Particle as PdgDatabase
+from particle.particle import enums
+
+from expertsystem.data import (
+    Parity,
+    Particle,
+    ParticleCollection,
+    QuantumState,
+    Spin,
+)
+
+
+# cspell:ignore pdgid
+def __get_lepton_qn(pdg_particle: PdgDatabase) -> tuple:
+    electron_lepton_number = 0
+    muon_lepton_number = 0
+    tau_lepton_number = 0
+    if pdg_particle.pdgid.is_lepton:
+        if "e" in pdg_particle.name:
+            electron_lepton_number = pdg_particle.pdgid / abs(
+                pdg_particle.pdgid
+            )
+        if "mu" in pdg_particle.name:
+            muon_lepton_number = pdg_particle.pdgid / abs(pdg_particle.pdgid)
+        if "tau" in pdg_particle.name:
+            tau_lepton_number = pdg_particle.pdgid / abs(pdg_particle.pdgid)
+
+    return electron_lepton_number, muon_lepton_number, tau_lepton_number
+
+
+def __get_hadron_qn(
```

Comment:
I try to reserve the term `get` for functions that don't do more than 'getting', and use something like `compute` or `create` to signal when a function does a computation.